src/app/courses/pages/new-course-page/new-course-page.component.scss

```diff
@@ -16,14 +15,33 @@
     padding-bottom: 35px;
     font-size: 14px;
   }
-  &__input,
-  &__password {
+
+  &__title {
+    font-size: 24px;
+    margin-bottom: 24px;
+  }
+
+  &__input {
```

Comment:
Add `padding-left` to the input. According to the mockup, the text inside the input shouldn't be so close to the border.